float64 ct
float64 gpa
float64 jt